bugguide/bugguide
# #62 Update description and added help menu links

### Description

Now it is much easier to find bugguide utilities by going to /admin/config

![](https://user-images.githubusercontent.com/1582129/103415069-925c8000-4b4e-11eb-9d77-dabb255f7f75.png)

### Testing steps

*   [ ] Log in as administrator
*   [ ] Go to /admin/config
*   [ ] You should see Bugguide section with all the links OR by hovering your mouse into Configuration and going to Bugguide you can see the available bugguide utilities
